### ISSUE TITLE
XSLT Support?

### ISSUE THREAD

**Issue by jlafitte** [emmetio/Emmet.codaplugin#29]
I haven't tried to use emmet with Coda in a while.  I see it's got a commercial license now.

I work with XSLT a lot, but the new version doesn't seem to work in my xsl file.  Does it no longer support it?  Is it now only a "Full Version" feature?  Or do I need to configure it somehow?

**Comment by sergeche** [comment]
Don’t see XSL syntax in latest Coda 2. New Emmet should work with XSL as well in free version

**Comment by jlafitte** [comment]
Well coda handles it like XML.  Let me rephrase what I had here before.  I have tried putting "xml":"xslt" and "xml":"xsl" in the syntaxProfiles.json, it didn't seem to work.

**Comment by jlafitte** [comment]
Nevermind.  I realized now what I'm trying to use is abbreviation, which looks like it is paid feature.  Sorry to bother you.
